### DISCUSSION TITLE
Open in new tab from card /detail layout search result

### DISCUSSION THREAD

**Discussion by rooobeert** [microsoft-search/pnp-modern-search#1934] [Ideas]
**Is your feature request related to a problem? Please describe.**

**This only applies to PDFs!** 

When the search results webpart is set up as a list and you click on the link of an item, it will open in a new tab by default. The details aswell as the card layout don´t open in a new tab.

If users now click on the link to open the pdf, it will open in the same tab. 
Problem: When closing the pdf at the top right (from the internal pdf viewer in SPO) the users are left at the document library, in which the document is stored. This is really confusing for some users, as they expect to be redirected to the search results.

**Describe the solution you'd like**
All search results, regardless of the layout option, should either open all in a new tab by default or at least have the option to toggle on or off. 

**Describe alternatives you've considered**
Customizing the card layout. But that didn´t seem to work or I used the wrong syntax.

**Additional context**
There have been multiple entries so far for that feature. But none of them seem to make it far. 
https://github.com/microsoft-search/pnp-modern-search/discussions/1452
https://github.com/microsoft-search/pnp-modern-search/issues/1451
https://github.com/microsoft-search/pnp-modern-search/issues/1457
https://github.com/microsoft-search/pnp-modern-search/issues/981

**Comment by wobba** [comment]
Not sure of your setup, but you can control which managed property you use as the click URL for pdf's to ensure you get the behavior you desire. We might change the internal logic to get a clickable URL and make it cover more scenarios better - but it's a bit complex.

**Comment by rooobeert** [comment]
Can you maybe give an example on how to configure the "click URL" to open in a new tab? Im not sure how to implement this.

**Reply by wobba** [reply]
This should open in a new tab:  `<a href="" target="_blank" data-interception="off">title</a>`

From https://docs.microsoft.com/en-us/sharepoint/dev/spfx/hyperlinking

**Comment by rooobeert** [comment]
Mhm, okay that doesn´t seem to work with the most of the card fields, since they do not allow HTML Code. 
I tried with Tags or Location, but it will always put the URL of the current site in there, where the search results is located.

**Reply by wobba** [reply]
We cannot easily control what 3p components do, as SharePoint is a not a direct scenario for them.

**Reply by JayMcW** [reply]
I have the same issue. The fields, like the Title field in the card layout, don't allow HTML.  You're stuck with the [out-of-the-box formatting](https://microsoft-search.github.io/pnp-modern-search/usage/search-results/layouts/), which for some reason, treats pdfs differently than .docx or .xlsx.
